File: Classes/Paciente.php
<?php
class Paciente extends Usuario{
    
    private $exames;
    private $resultados;
    private $historicoMecido;
    private $alunoResponsavel; 

    public function __construct($nome, $id, $dataNascimento, $cpf, $email, $endereco, $exames, $resultados,$historicoMecido,$alunoResponsavel){
      parent:: __construct($nome, $id, $dataNascimento, $cpf, $email, $endereco);

      $this->exames = $exames;  
      $this->resultados = $resultados;
      $this->historicoMecido = $historicoMecido;
      $this->alunoResponsavel = $alunoResponsavel;
   }

    public function getExames(){
        return $this->exames;
    }
    
    public function setExames($exames){
        $this->exames = $exames; 
    }

    public function getResultados(){
        return $this->resultados;
    }
    
    public function setResultados($resultados){
        $this->resultados = $resultados; 
    }

    
    public function getHistoricoMecido(){
        return $this->historicoMecido;
    }
    
    public function setHistoricoMecido($historicoMecido){
        $this->historicoMecido = $historicoMecido; 
    }
  
    public function getAlunoResponsavel(){
        return $this->alunoResponsavel;
    }
    
    public function setAlunoResponsavel($alunoResponsavel){
        $this->alunoResponsavel = $alunoResponsavel; 
    }

    public function __toString(){
        return "Exane: {$this->exames} - Resultado: {$this->resultados} - Historico Medico: {$this->historicoMecido} - Aluno Responsavel: {$this->alunoResponsavel}";
    }

  }
?>

File: PastaJava/Classe/Paciente.java
public class Paciente extends Pessoa {
    private String exames;
    private String resultados;
    private String historicoMedico;
    private String alunoResponsavel;

    public Paciente (){}
   
    public Paciente(String nome, int id, String cpf, String exames, String resultados, String historicoMedico, String alunoResponsavel) {
        super(nome, id, cpf);
        this.exames = exames;
        this.resultados = resultados;
        this.historicoMedico = historicoMedico;
        this.alunoResponsavel = alunoResponsavel;
    }

    public String getExames() {
        return exames;
    }

    public void setExames(String exames) {
        this.exames = exames;
    }

    public String getResultados() {
        return resultados;
    }

    public void setResultados(String resultados) {
        this.resultados = resultados;
    }

    public String getHistoricoMedico() {
        return historicoMedico;
    }

    public void setHistoricoMedico(String historicoMedico) {
        this.historicoMedico = historicoMedico;
    }

    public String getAlunoResponsavel() {
        return alunoResponsavel;
    }

    public void setAlunoResponsavel(String alunoResponsavel) {
        this.alunoResponsavel = alunoResponsavel;
    }

    @Override
    public String toString() {
        return super.toString() + " Paciente [exames=" + exames + ", resultados=" + resultados + ", historicoMedico=" + historicoMedico + ", alunoResponsavel=" + alunoResponsavel + "]";
    }

}
